packages/react/src/hooks/internal/useBrowserTabsChannel.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react';
+
+export const useBrowserTabsChannel = <T = unknown>({
+  channelName,
+  onMessage,
+}: {
+  channelName: string;
+  onMessage: (args: T) => void;
+}) => {
+  const [tabsChannel] = useState(new BroadcastChannel(channelName));
+
+  const postMessage = (data: T) => {
+    tabsChannel.postMessage(data);
+  };
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** this function might still fail in RN, it's exposed and used when the ws event comes